### ISSUE TITLE
Fuse batch norm into conv

### ISSUE THREAD

**Issue by odjuricicTT** [tenstorrent/tt-mlir#4155]
From torch xla we get ttir.batchnorm. For inference it can be fully fused into the preceding convolution.

https://medium.com/@sim30217/fusing-convolution-with-batch-normalization-f9fe13b3c111

There are two approaches to this tho:
1. Decompose batch norm and then fuse individual ops
2. Fuse batchnorm in one step (this requires decomposition tho)

**Comment by mvasiljevicTT** [comment]
Merged both PRs regarding this
